dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ExecutorServiceImpl.java

```diff
@@ -127,6 +128,7 @@
      * @return execute process instance code
      */
     @Override
+    @Transactional(rollbackFor = RuntimeException.class)
```

Comment:
Maybe remove `rollbackFor = RuntimeException.class` is better? What do you think?

Reply:
I agree that. And push another commit to remove it.